### ISSUE TITLE
"View source data" pop-up window not scrollable on mobile.

### ISSUE THREAD

**Issue by wafels** [Helioviewer-Project/helioviewer.org#474]
For features and events on mobile, sometimes the "View source data" pop-up window is not scrollable on a mobile device (I tested this on an iPhone/Safari. This means that the GIFs that are now included for DONKI CME notifications that have modeling content are not visible. They should be visible.

**Comment by dgarciabriseno** [comment]
Yep, this is WIP. It's scrollable on beta right now, but the offsets are wrong so some text is still not viewable. Working on it.

**Comment by dgarciabriseno** [comment]
@wafels Take a look now.
Also touching/clicking on the gifs will focus them in the middle of the screen.

**Comment by wafels** [comment]
Ok, thanks!
